templates/stream.app.navigation.php

```diff
@@ -53,6 +53,11 @@
 				<input id="feed-link" class="feed-link" type="text" readonly="readonly" value="<?php p($_['rssLink']); ?>" />
 				<a class="icon-clippy" data-clipboard-target="#rssurl input"></a>
 			</span>
+			<div id="activity-personal-settings-link">
+				<a href="<?php p($_['personalSettingsLink']); ?>">
+					<span class="no-icon"><?php p($l->t('Personal Activity Settings') . ' ↗'); ?></span>
```

Comment:
I think we only use the `↗` for external links, but pinging @jancborchardt to clarify on that.

Reply:
We use either the [icon-external](https://docs.nextcloud.com/server/latest/developer_manual/design/icons.html?highlight=external) or ↗ for external links. I think it's better not to use any icon here or maybe the [activity app icon](https://github.com/nextcloud/activity/blob/master/img/activity-dark.svg).

Reply:
Done. This is is how it looks without `↗`:
![image](https://user-images.githubusercontent.com/42591237/126864880-30be18b0-e609-4422-806a-6473ec52e180.png)